projects/vdk-core/src/vdk/internal/util/utils.py

```diff
@@ -40,3 +50,27 @@ def parse_config_sequence(
     if sequence:
         sequence = [i.strip() for i in sequence.split(sep)]
     return sequence if sequence else []
+
+
+def exit_with_error(user_error: bool, log: Logger, exception: Exception):
```

Comment:
https://wiki.c2.com/?UseEnumsNotBooleans

we have an enum for that: https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/src/vdk/internal/core/errors.py#L55